app/src/main/java/eu/neilalexander/yggdrasil/DnsActivity.kt

```diff
@@ -127,7 +127,7 @@ class DnsActivity : AppCompatActivity() {
                     view.findViewById<ImageButton>(R.id.deletePeerButton).tag = i
 
                     view.findViewById<ImageButton>(R.id.deletePeerButton).setOnClickListener { button ->
-                        val builder: AlertDialog.Builder = AlertDialog.Builder(this)
+                        val builder: AlertDialog.Builder = AlertDialog.Builder(ContextThemeWrapper(this, R.style.Theme_MaterialComponents_DayNight_Dialog))
```

Comment:
I don't think that this solution is optimal. The core problem is that colorPrimary in themes.xml is set to white. Buttons are already exist, but their text color is white.

I think the best solution is to change primary color in theme.